nn-specials/src/main/java/org/frankframework/extensions/esb/EsbJmsListener.java

```diff
@@ -201,6 +204,15 @@ protected String getResultFromxPath(String message, String xPathExpression) {
 		return found;
 	}
 
+	private static TransformerPool getXPathTransformerPool(String xPathExpression) throws TransformerConfigurationException {
+		if (LOG_KEY_TRANSFORMER_POOLS.containsKey(xPathExpression)) {
+			return LOG_KEY_TRANSFORMER_POOLS.get(xPathExpression);
+		}
+		TransformerPool xpathTransformerPool = TransformerPool.getUtilityInstance(XmlUtils.createXPathEvaluatorSource("", xPathExpression, OutputType.TEXT, false), XmlUtils.DEFAULT_XSLT_VERSION);
+		LOG_KEY_TRANSFORMER_POOLS.put(xPathExpression, xpathTransformerPool);
+		return xpathTransformerPool;
+	}
```

Comment:
eingelijk moet dit in de configure toch? het maken van de pools (en gooien van een fout wanneer er eentje is), dan hoeft hij ook niet static te zijn

Reply:
De pools moeten wel gecacht worden want ze worden voor iedere message aangeroepen, maar je hebt gelijk dat het misschien niet static hoeft.
De log keys kunnen komen uit de configuration, of uit deployment.properties, of uit appconstants.properties.

Omdat er globale keys zijn dacht ik dat het static kon, dan wordt er meer gedeeld.

Ik zou ze wel in de configure kunnen maken.